CHANGELOG

```diff
@@ -1,3 +1,31 @@
+# CBMC 6.3.0
+
+This release addresses build failures on aarch64 (64-bit ARM) platforms (via PR #8366) and for some CMake configurations (via PR #8435). Users of loop invariants with dynamic frames have two changes to their user experience:
+1) Users will no longer need to give unwinding specifications for `do { ... } while(0)` loops.
+2) Loop invariants that are conjunctions will be turned into more fine-grained properties to ease debugging of loop invariants when they aren't successfully proved.
+
+## What's Changed
+* Contracts: always remove spurious do {... } while(0) loops by @tautschnig in https://github.com/diffblue/cbmc/pull/8459
```

Comment:
Tell the user why this is great... e.g. "Therefore, introducing a do { ... } while (0) loop, either explicitly or as the result of a macro, will not change the numbering of other loops, and therefore will not impact the setting of unwindsets for other loops."

Reply:
Or something like that...